packages/test-utils/src/index.ts

```diff
@@ -189,16 +221,18 @@ function _keystoneRunner(adapterName, tearDownFunction) {
   };
 }
 
-function _before(adapterName) {
-  return async function (setupKeystone) {
+function _before(adapterName: AdapterName) {
+  return async function (
+    setupKeystone: (adapterName: AdapterName) => Promise<{ keystone: Keystone<string>; app: any }>
+  ) {
     const { keystone, app } = await setupKeystone(adapterName);
     await keystone.connect();
     return { keystone, app };
   };
 }
 
-function _after(tearDownFunction) {
-  return async function (keystone) {
+function _after(tearDownFunction: () => Promise<any> | void) {
```

Comment:
```suggestion
function _after(tearDownFunction: () => Promise<void> | void) {
```

Reply:
I had this initially, but some of our tear down functions don't necessarily return void, so changed to any out of laziness. I think void is the right type though, I'm going to go in and fix those tear down functions to return void rather than arbitrary promises 👍

Reply:
void is a bit of a weird type because essentially anything is assignable to it in some cases like below(but it's still useful over any because it conveys that the return value should be ignored)

```ts
function something(cb: () => void) {}

something(() => '')
```